src/backend/src/migrations/1667891146340-ppq-iteration-two.ts

```diff
@@ -0,0 +1,71 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class PpqIterationTwo1667891146340 implements MigrationInterface {
+  name = 'ppqIterationTwo1667891146340';
+
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(`ALTER TABLE "ppq" DROP COLUMN "name"`);
+    await queryRunner.query(`ALTER TABLE "ppq" DROP COLUMN "email"`);
+    await queryRunner.query(`ALTER TABLE "ppq" DROP COLUMN "branch"`);
+    await queryRunner.query(`ALTER TABLE "ppq" DROP COLUMN "initiative_name"`);
+    await queryRunner.query(
+      `ALTER TABLE "ppq" DROP COLUMN "initiative_description"`,
+    );
+    await queryRunner.query(`ALTER TABLE "ppq" DROP COLUMN "data_elements"`);
+    await queryRunner.query(
+      `ALTER TABLE "ppq" DROP COLUMN "has_sharing_of_personal_information"`,
+    );
+    await queryRunner.query(
+      `ALTER TABLE "ppq" DROP COLUMN "contains_personal_information"`,
+    );
+    await queryRunner.query(
+      `ALTER TABLE "ppq" ADD "title" character varying NOT NULL`,
+    );
+    await queryRunner.query(
+      `ALTER TABLE "ppq" ADD "description" character varying NOT NULL`,
+    );
+    await queryRunner.query(
+      `ALTER TABLE "ppq" ADD "delegated_review_type" character varying`,
+    );
+    await queryRunner.query(`ALTER TABLE "ppq" ADD "has_data_linking" boolean`);
+    await queryRunner.query(
+      `ALTER TABLE "ppq" ALTER COLUMN "pia_type" SET NOT NULL`,
+    );
```

Comment:
Seeing it more of a comment than a question.

Right. For context - there are two migrations - first to get rid of the existing data, followed by other to edit the column and it's types to support version-2 of the PPQ design.

Also, if you're concerned about the migrations per say, note that this is tool managed and if you're happy with the `ppq.entity.ts` file. It should be in place correctly.

However, you're free to connect to the environment specific database for the implementation specifics once merged as well.

Marking it resolved. [Unresolve if otherwise]